code/modules/ninja/energy_katana.dm

```diff
@@ -0,0 +1,10 @@
+
+
+/obj/item/weapon/katana/energy
+	name = "energy katana"
+	desc = "a katana infused with a strong energy"
+	icon_state = "energy_katana"
+	item_state = "energy_katana"
+	force = 40
+	throwforce = 20
+	slot_flags = SLOT_BELT
```

Comment:
The definition of ```slot_flags = SLOT_BELT``` was to get rid of the back slot, but sure I guess, I'll add it back. (I disagreed with it for an ENERGY katana, but sure).